### PR TITLE
Fix python syntax and remove pipenv dependency

### PR DESCRIPTION
**Pull request by nc-yehs**

### CODE REVIEW

**Comment by nc-firthh** [comment]
@nc-yehs While you're here, mind updating the readme to remove pip and pipenv from the dependnecies in the Python challenge setup instructions?

File: README.md [hunk 1]
```diff
@@ -144,17 +144,15 @@ Run `./gradlew test` to run the test suite
 
 ### Dependencies
 
-* Python 3
-* [pip](https://pip.pypa.io/en/stable/installing/)
+* Python 3.7 or later (run `python3 --version` to check your version)
 
-### Installation
+### Setup
```
Comment (nc-firthh) [inline README.md:149]: NP: For consistency, I think I'd prefer this stay `Installation` or that the rest of README instructions also get updated to say `### Setup`.

I don't have a preference between the two, I just think it should be the same everywhere.

Comment (nc-yehs) [reply README.md:149]: the javascript one uses `Setup` too, I'll change them all to Setup

Comment (nc-firthh) [reply README.md:149]: Ah good catch, didn't see that!

**Comment by nc-firthh** [comment]
@nc-yehs I don't have write access to this repo, so I can't approve 😞 

@pattertom, are you good approving this fork PR? Or would you rather this be moved to a branch PR?

**Review by pattertom — APPROVED** [review]
💸  looks good to me

**Pull request merged**